src/main/java/gmbs/domain/WinningResult.java

```diff
@@ -0,0 +1,29 @@
+package gmbs.domain;
+
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+
+public class WinningResult {
+    private final Map<Ranking, Integer> winningResult = new EnumMap<>(Ranking.class);
+
+    public WinningResult(List<Ranking> rankings) {
+        putValues(rankings);
+    }
+
+    public long calculatePrizeSum() {
+        return winningResult.entrySet().stream()
+                .mapToLong(entry -> entry.getKey().multiple(entry.getValue()))
+                .sum();
```

Comment:
```suggestion
        return winningResult.entrySet()
                .stream()
                .mapToLong(entry -> entry.getKey().multiple(entry.getValue()))
                .sum();
```
체이닝의 `.`은 한 줄에 하나씩만 찍어용~